client/js/components/map/publicdetail/diplankarte/DiplanKarteWrapper.vue

```diff
@@ -157,6 +140,127 @@ const handleDrawing = (event) => {
   emit('locationDrawing', payload)
 }
 
+// Feature: Set location info
+
+const closeLocationInfo = () => {
+  isLocationInfoClosed.value = true
+}
+
+const isLocationInfoClosed = ref(false)
+
+const isLocationToolSelected = computed(() => {
+  return store.state.PublicStatement.activeActionBoxTab === 'draw'
```

Comment:
Here and below: A long time ago (so you may not know this), we decided to always use `mapState`, `mapGetters`, etc. instead of accessing the store as you did here